src/components/download-links/style.scss

```diff
@@ -1,7 +1,8 @@
 download-links {
   position: relative;
-  margin-left: auto;
+  margin-left: 0;
   display: flex;
+  border-left: 0 !important;
```

Comment:
Actually.... this is completely unnecessary, was fixed by `margin-left: 0` and was an experiment. Removing with another incoming commit!